### DISCUSSION TITLE
Can't download photos

### DISCUSSION THREAD

**Discussion by gomme600** [photoprism/photoprism#1066] [Q&A] [answered]
Hi,
I am unable to download any photos, chrome gives an error "acces denied".
Thanks for any help.
![Capture_photoprism_error](https://user-images.githubusercontent.com/35890638/106388480-fb017b80-63de-11eb-8077-160064037a7c.JPG)

**Comment by lastzero** [comment]
Did you restart the server while using the app? In that case, the link may not be valid anymore. Or your local download folder is not writeable by your browser.

**Comment by gomme600** [comment]
No, I opened the photo in my web browser and immediately pressed the download button. It has done this from the very beginning (over a month). The server has been restarted multiple times and I have even started again from scratch. My download folder is fine as downloads from other sites work fine. I just checked and it works fine on my LAN using the internal IP. Using my hostname doesn't work though. Maybe it's a NGINX problem or a configuration problem with PhotoPrism? Not sure...

**Comment by andrewlow** [comment]
Hostname issues may be something funky with the DNS mapping - or nginx configuration. 

Start by validating that the hostname == ip address. This is probably the problem.

If the hostname is the same IP - then nginx is doing something funky to your query based on the name (vs. IP if you still go through nginx for IP based access)

Since the IP is working - then this isn't a photoprism issue, it's about routing/naming. I would suggest you close this issue.

**Comment by lastzero** [comment, marked as answer]
I'd investigate potential issues with the nginx proxy. Here are our related docs:

https://docs.photoprism.org/getting-started/proxies/nginx/
